pep-8104.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 8104
+Title: 2023 Term steering council election
+Version: $Revision$
+Last-Modified: $Date$
+Author: Ee Durbin <ee@python.org>
+Sponsor: Brett Cannon <brett@python.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 08-Nov-2022
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the December
+2022 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2023 term
+(i.e. Python 3.12).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period shall be: November 14, 2022 through November 29, 2022 12:00 UTC
```

Comment:
```suggestion
The nomination period shall be: November 14, 2022 through `November 29, 2022 12:00 UTC
<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+nominations+close&iso=20221129T12>`_
```

Its helpful to link this time, so the viewer can convert it to their own time zone...even with AoW, people still mess this up. Of course, there's other such websites to choose from for this, I merely suggested on.

Reply:
Settled this by addressing @brettcannon's request to go with AoE and not specify a time for start. We'll start given periods at approximately 12:00 UTC on the stated day.